lib/elixir/test/elixir/registry_test.exs

```diff
@@ -992,6 +992,20 @@ defmodule RegistryTest do
     assert {%RuntimeError{message: "some error"}, _stacktrace} = error
   end
 
+  test "send works", %{registry: registry} do
+    name = {registry, "self"}
+    Registry.register_name(name, self())
+    Registry.send(name, :message)
+    assert_received :message
+  end
+
+  test "send works with value", %{registry: registry} do
+    name = {registry, "self", "value"}
+    Registry.register_name(name, self())
+    Registry.send(name, :message)
```

Comment:
Instead of send, maybe the test can use GenServer.cast? The message will be wrapped but that’s fine.

Reply:
Updated, didn't bother to create a GenServer stub, still asserts we receive the expected message.